src/cryptography/x509.py

```diff
@@ -190,6 +190,9 @@ def __iter__(self):
     def __len__(self):
         return len(self._attributes)
 
+    def __repr__(self):
+        return repr(self._attributes)
```

Comment:
haha, yeah I was fixing that as soon as I uploaded it. I should swap it to 0!r though. Will do that momentarily.